content/changelog/2024-04-19.md

```diff
@@ -0,0 +1,168 @@
+
+<div style={{ position: 'relative', top: '-70px' }} id="protected-branches"></div>
+### Protected branches
+
+Users of the Neon [Scale](/docs/introduction/plans#scale) plan can now designate a branch as "protected". This status restricts branch access based on IP addresses &#8212; only IPs on your project’s allowlist can access a protected branch. Typically, branches containing production or sensitive data are designated as protected.
+
+For details on configuring a protected branch, please refer to our [Protected branches](/docs/guides/protected-branches) guide.
+
+<div style={{ position: 'relative', top: '-70px' }} id="large-database-support"></div>
+### Large database support (beta)
+
+We are currently testing architecture changes to support large databases from 300 GiB to 2 TiB. Access to this feature is limited to paying users by request. If you want to try it out, you can request from the **Beta** tab under **Project Settings**, or use this direct link:
+
+- [Request support for large database](http://console.neon.tech/app/projects?modal=request_large_db)
+
+<div style={{ position: 'relative', top: '-70px' }} id="neon-serverless-driver-jsr-package"></div>
+### Neon Serverless driver JavaScript Registry (JSR) package
+
+The Neon serverless driver is now available as a [JavaScript Registry (JSR)](https://jsr.io/docs/introduction) package.
+
+```bash
+@neon/serverless
+```
+
+The JavaScript Registry (JSR) is a package registry for JavaScript and TypeScript. JSR works with many runtimes (Node.js, Deno, browsers, and more) and is backward compatible with `npm`. For example, to install using `npm`:
+
+```bash
+npm install @neon/serverless
+```
+
+For a deeper dive, check out this blog post announcing support for this feature: [Neon Serverless Driver on JSR](https://neon.tech/blog/neon-serverless-driver-on-jsr)
+
+### Fixes & improvements
+
+- You can now view the last active time for a compute endpoint associated with your branch, which is useful for determining which branches you have accessed recently. In the Neon Console, you can view the **Last active** time on the **Branches** page or in the **Computes** section on individual branch pages. Additionally, the Neon API [Get a compute endpoint](https://api-docs.neon.tech/reference/getprojectendpoint) method response now includes a `last_active` field.
+- The Neon SQL Editor now uses the Neon serverless driver for database connections. If you utilize the Neon [IP Allow](/docs/introduction/ip-allow) feature and your public IP address isn't currently on the allowlist, you may encounter a pop-up when you next query your database through the Neon SQL Editor. This pop-up requests permission to add your IP address to the allowlist.
+- IPv6 addresses are now truncated to the `/64` subnet to minimize the risk of DDoS attacks. This change limits potential abuse by reducing the usable address space while maintaining sufficient network functionality for legitimate users.
```

Comment:
```suggestion
```
This change is for a ddos protection and not something I think needs to be in the changelog